### DISCUSSION TITLE
键盘控制

### DISCUSSION THREAD

**Discussion by streesobrs** [BYJRK/DotNet-Discussions#121] [WPF 开发]
页面分为两部分，左边是用listbox做的一个导航栏，右边是Frame控件用来显示其他页面的
点击listbox其中一个按钮后，切换到一个window2的页
问题来了：
写了键盘↑←↓→的事件，按了键盘上的键，控制的却是左边listbox中的按钮，设了焦点但还是控制的listbox的按钮，怎么才能让他只控制window2页

**Comment by BYJRK** [comment]
你可以将 `ListBox` 以及 `ListBoxItem` 的 `Focusable` 设为 `false`
另外，你的按键是哪个控件在响应？
